packages/amplify-cli/src/extensions/amplify-helpers/root-stack-status.ts

```diff
@@ -0,0 +1,36 @@
+import { pathManager } from 'amplify-cli-core';
+import { nestedStackFileName } from 'amplify-provider-awscloudformation';
+import { hashElement, HashElementOptions } from 'folder-hash';
+import * as fs from 'fs-extra';
+
+export function getHashForRootStack(dirPath, files?: string[]) {
+  const options: HashElementOptions = {
+    folders: { exclude: ['.*', 'node_modules', 'test_coverage', 'dist', 'build'] },
```

Comment:
the build will have the cfn.json file..so should this folder be excluded?

Reply:
this for build inside build folder when we apply cfnmodifiers

Reply:
but including this would exclude the top level build and the inner build folder correct?

Reply:
I am getting root stack changes inside build also

Reply:
do a build when there is a difference in override.ts file